roles/precis/tasks/main.yml

```diff
@@ -135,3 +141,19 @@
     - "{{ precis_event_listener_service_name }}"
     - "{{ precis_endpoint_service_name }}"
     - "{{ precis_generator_service_name }}"
+
+- name: Configure precis cleanup docker bash wrapper.
+  template:
+    src: precis_cleanup.j2
+    mode: a+x
+    dest: /usr/local/sbin/precis_cleanup
+
+- name: Set up precis cleanup cron job.
+  cron:
+    name: "precis_cleaup"
+    minute: "0"
+    hour: "0"
+    day: "1"
+    month: "*"
+    weekday: "*"
+    job: "/usr/local/sbin/precis_cleanup | /usr/bin/logger -t \"precis_cleanup\""
```

Comment:
Please import the `cron` role when installing a cron job. It can be done like this:  https://github.com/ChameleonCloud/chi-in-a-box/blob/master/roles/mariadb/tasks/main.yml#L2-L6

That task will ensure a special script is installed on the target box, and store the location to that script as `cron_notification`. Then your cron `job` parameter becomes:

```yaml
job: "/usr/local/sbin/precis_cleanup | /usr/bin/logger -t precis_cleanup && {{ cron_notification }}"
```

The notification script is responsible for reporting a metric (last successful run) to Prometheus, which in turn allows us to detect when jobs start failing for some reason.